pkgs/applications/virtualization/nvidia-container-toolkit/default.nix

```diff
@@ -79,7 +95,7 @@ buildGoModule rec {
     substituteInPlace $out/etc/nvidia-container-runtime/config.toml \
       --subst-var-by glibcbin ${lib.getBin glibc}
 
-    ln -s $out/bin/nvidia-container-{toolkit,runtime-hook}
+    ln -s $out/bin/nvidia-container-runtime-hook $out/bin/nvidia-container-toolkit
```

Comment:
Would be nice (not a blocker) to add a comment about what's this used by? Where do we fail if we just remove the line entirely?

Reply:
AFAIU this is part of the "expected output files". I've added a link to the original script that uses this line.